web/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
hey switch to a stable flutter channel and then make a new PR looks like you are using Flutter Beta Version